rosidl_generator_cpp/resource/msg__builder.hpp.em

```diff
@@ -0,0 +1,80 @@
+@# Included from rosidl_generator_cpp/resource/idl__builder.hpp.em
+@{
+message_typename = '::'.join(message.structure.namespaced_type.namespaced_name())
+}@
+
+@[for ns in message.structure.namespaced_type.namespaces]@
+namespace @(ns)
+{
+
+@[end for]@
+@[if not message.structure.members]@
+::@(message_typename) build_@(message.structure.namespaced_type.name)()
```

Comment:
@dirk-thomas weren't we dropping `*::build_*()` functions in favour of `*::build<>()`?

Reply:
Actually this block could be removed. `message.structure.members` is always a non-empty list.

Reply:
Done in 1e0e96ebef47ef9ad154487702899f6e82bb4297.